backend/dining/api_wrapper.py

```diff
@@ -71,19 +58,15 @@ def request(self, *args, **kwargs):
 
     def get_venues(self):
```

Comment:
so this is called every time the /dining/venues/ endpoint is hit. not sure how often this is hit by Mobile, but since we're already doing a cronjob on the menu, do we want to cronjob this so that we can avoid doing this clean-up and extra API call everytime?

Reply:
Oh shoot yes I'm down I'll put this in the shortcut